src/base/static/utils/config-loader-utils.js

```diff
@@ -15,7 +15,9 @@ const transformStoryContent = storyConfig => {
             url: chapter.url,
             zoom: chapter.zoom || storyContent.default_zoom,
             hasCustomZoom: !!chapter.zoom,
-            panTo: chapter.panTo || null,
+            panTo: chapter.panTo
+              ? [chapter.panTo.lng, chapter.panTo.lat]
```

Comment:
Why are we wrapping this in an array? Is it possible for the config to define `panTo in a way that we don't have to keep mutating it?

It creates yet another representation of the same thing...

Reply:
Yeah, we can change that section of the config so it's just keyed in as an array.